modules/apps/foundation/contacts/contacts-service/src/main/java/com/liferay/contacts/service/impl/EntryServiceImpl.java

```diff
@@ -34,18 +38,55 @@
 	@Override
 	public JSONArray searchUsersAndContacts(
 			long companyId, String keywords, int start, int end)
-		throws PortalException {
+		throws Exception {
```

Comment:
Hey @ChrisKian, which change necessitated that this be an Exception?  I think we can keep it a PortalException, but I want to make sure I'm not missing anything.

Reply:
Ah, I see - it's the `PermissionCheckerFactory.create(user)` call that Jonathan changed later.  That's why my IDE didn't complain about me reverting it back to a PortalException.

Reply:
Bingo.  Sorry for the confusion!

Reply:
np :)